uwmmss0348.xml

```diff
@@ -38,18 +37,19 @@
 	<archdesc level="collection" relatedencoding="MARC21">
 		<did>
 			<physloc type="locationcode" audience="internal">wimiugma</physloc>
-			<unittitle encodinganalog="245$a">
-				<!--Student Wisconsin Education Association Records -->
-				<unitdate type="inclusive" encodinganalog="245$f" normal="[ie. 1917/1940]"
-					><!--2000  to 2011--></unitdate>
+			<unittitle encodinganalog="245$a"> Student Wisconsin Education Association at
+				UW-Milwaukee Records, <unitdate type="inclusive" encodinganalog="245$f"
+					normal="2000/2011">2000-2011</unitdate>
```

Comment:
I should have mentioned that the normal attribute also had to be updated. My fault. Commenting here for posterity. Do make sure to remove the comments (highlight the line, then ctrl-shift-comma) when you're actually filling in the template.